examples/dtls-echo/keys/tinydtls_keys.h

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
If you add a `keys` directory, then I would expect 2 include files there: one for the psk keys, one for the ECC keys. Each one would be included conditionally (using #ifdef DTLS_PSK, etc) in the c files.
The second solution is to keep the header file like it is now, but put it at the same level as the C files.